vayesta/ewf/fragment.py

```diff
@@ -306,7 +306,9 @@ def get_solver_options(self, solver):
         has_actspace = (
             (solver == "TCCSD")
             or ("CCSDt'" in solver)
+            or ("CCSDt" in solver)
```

Comment:
`"CCSDt'" in solver` implies `"CCSDt" in solver` (acceptable if you want to be very explicit though)